packetbeat/docs/fields.asciidoc

```diff
@@ -1439,21 +1439,21 @@ The full collection name. The full collection name is the concatenation of the d
 
 ==== mongodb.numberToSkip
 
-type: number
```

Comment:
@tsg Type number does not exist in elasticsearch. I made the assumption that this should be an integer. Is this correct?